Umbraco.Docs.Samples.Web/Backoffice Search/BackofficeSearchComposer.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Umbraco.Cms.Core.Composing;
+using Umbraco.Cms.Core.DependencyInjection;
+using Umbraco.Cms.Infrastructure.Examine;
+using Umbraco.Extensions;
+
+//https://our.umbraco.com/documentation/Extending/Backoffice-Search/
+
+namespace Umbraco.Docs.Samples.Web.BackofficeSearch
+{
+    public class BackofficeSearchComposer : IComposer
```

Comment:
Ideally, this would be changed from Composer to using a builder extension method. (IComposer is only for packages) Simular to https://github.com/umbraco/Umbraco.Docs.Samples/blob/develop/Umbraco.Docs.Samples.Web/Trees/UmbracoBuilderSearchableTreeExtensions.cs